components/raftstore-v2/src/fsm/apply.rs

```diff
@@ -126,6 +128,7 @@ impl<EK: KvEngine, R: ApplyResReporter> ApplyFsm<EK, R> {
                 }
             }
             self.apply.flush();
+            STORE_APPLY_LOG_HISTOGRAM.observe(duration_to_sec(received_time.saturating_elapsed()));
```

Comment:
This is not accurate as the fsm may be reschedule where there is an `await`.

Reply:
From end-to-end perspective, I think it's the time the user request actually pays for the apply. So it's ok.

Reply:
No, it's not. It just the time apply finish a round of processing. Before the timer stops, user may have sent and received multiple round of req/resp.

Reply:
Got it, I will using water fall to replace removing  this metrics in next pr.